Assets/GothicVR/Scripts/Manager/GvrSceneManager.cs

```diff
@@ -166,14 +166,12 @@ private async Task ShowLoadingScene(string worldName = null)
             await Task.Delay(ensureLoadingBarDelayMilliseconds);
         }
 
-        private void SetLoadingTextureForWorld(string worldName)
+        private void SetLoadingTextureForWorld(string worldName, bool newGame = false)
```

Comment:
You name newGame as parameter, but don't handle it inside this function. Are you missing something?

Reply:
did you mean use it? if so check line 174

Reply:
Too much beer. Sorry. Solved. 🤣